sdks/builds/llvm.mk

```diff
@@ -48,8 +48,10 @@ endif
 #  $(1): target
 define LLVMTemplate
 
+# -DLLVM_ENABLE_LIBXML2=Off is needed because xml2 is not used and it breaks 32-bit builds on 64-bit Linux hosts
 _llvm-$(1)_CMAKE_ARGS = \
-	$$(llvm-$(1)_CMAKE_ARGS)
+	$$(llvm-$(1)_CMAKE_ARGS) \
+	-DLLVM_ENABLE_LIBXML2=Off \
```

Comment:
Move that to a new line in between the following lines: https://github.com/mono/mono/blob/05bc22015c9cfedfb6faead1fe3aac7e918e498a/llvm/build.mk#L32-L33